crates/compiler/load_internal/src/file.rs

```diff
@@ -1342,7 +1342,31 @@ fn handle_root_type<'a>(
 
                 Ok((header_output, RootType::Module { main_path }))
             }
-            App { .. } | Package { .. } | Platform { .. } => Ok((header_output, RootType::Main)),
+            Package { .. } => {
```

Comment:
Seems better to just move this up to the above matched group of modules. Is that not an option?

Reply:
I can, just need another match to set the root type

Reply:
Done